input/pages/qi-elements.md

```diff
@@ -1,4 +1,3 @@
-<div class="new-content" markdown="1">
```

Comment:
Is there an end div that needs to be removed somewhere too?

Reply:
No end div, we're good